### PR TITLE
Implements Productive Community Conversation Generator to the game.

### PR DESCRIPTION
**Pull request by ThatFiveGuys**
## About the PR 
This pr implements an automatic productive community generator to the game. During setup the generator runs. The generator has a 45% chance of removing 1-3 of any obj child, a 25% chance of removing 1-2 reagents, a 20% chance of removing a job, and a 10% chance of porting a random tg pull request. These changes are automatically documented at the top of the changelog.

## Why's this needed? 
This should lead to many calm and productive discord and forums conversations. 

## Changelog

Since this automatically updates the changelog, a changelog should not be needed here.

### CODE REVIEW

**Comment by Eagletanker** [comment]
This is good lets do it

**Comment by frawhst** [comment]
wait no i want this

File: _std/__build.dm [hunk 1]
```diff
@@ -96,7 +96,7 @@ o+`        `-` ``..-:yooos-..----------..`
 //#define MAP_OVERRIDE_CONSTRUCTION		// Construction mode
 //#define MAP_OVERRIDE_DESTINY			// Destiny/RP
 //#define MAP_OVERRIDE_CLARION			// Destiny/Alt RP
-//#define MAP_OVERRIDE_COGMAP			// Cogmap
+#define MAP_OVERRIDE_COGMAP			// Cogmap More like Pogmap
```
Comment (frawhst) [inline _std/__build.dm:99]:
```suggestion
//#define MAP_OVERRIDE_COGMAP
```
simply not pog enough

**Comment by frawhst** [comment]
finally i won't have to nerf anything. the feedback will come on its own

**Pull request merged**